clang/lib/Headers/cuda_wrappers/new

```diff
@@ -91,12 +91,14 @@ __device__ inline void operator delete[](void *ptr,
 #endif
 
 // Device overrides for placement new and delete.
+#if _LIBCPP_STD_VER < 26
```

Comment:
This helps with libc++. 

Do we run into this issue with libstdc++, too? If so, we may check its version here, too.

Reply:
Looks like it
https://github.com/gcc-mirror/gcc/blob/9dd9a88b75334bc079b8ab5fb2dbb5d56765bd60/libstdc%2B%2B-v3/libsupc%2B%2B/new#L198-L214

I have an older version of libstdc++ on my system, so I can't easily test it, but it looks like changing this to
`#if _LIBCPP_STD_VER < 26 || __cpp_lib_constexpr_new < 202406L` should work.

Reply:
Yup. Reproducible on compiler explorer: https://godbolt.org/z/xr5nEhGnr

Reply:
Alright, both should be fixed now.